lib/translator/inquiry.cc

```diff
@@ -335,36 +335,37 @@ StatusCode InquiryToNvme(Span<const uint8_t> raw_scsi,
 
 // Main logic engine for the Inquiry command
 StatusCode InquiryToScsi(Span<const uint8_t> raw_scsi, Span<uint8_t> buffer,
-                         Span<const nvme::GenericQueueEntryCmd> nvme_cmds) {
```

Comment:
Passing `identify_ns` and `identify_ctrl` instead of the span to be consistent with the param list in `InquiryToNvme`

This also addresses the bug where `InquiryToNvme` had `identify_ns = nvme_cmds_[0]` but `InquiryToScsi` had `identify_ns = nvme_cmds_[1]`